.github/workflows/create-release.yml

```diff
@@ -29,57 +29,87 @@ env:
 
 
 jobs:
-  create-release:
+  generate-requirements-freeze:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Checkout spinalcordtoolbox (main branch)
+    - name: Checkout spinalcordtoolbox
       uses: actions/checkout@v2
-      with:
-        ref: ${{ env.MAIN_BRANCH }}
-        fetch-depth: 0
 
     - name: Install spinalcordtoolbox
       run: |
         ./install_sct -iy
 
-    # The GitHub Actions bot email was taken from: https://github.community/t/github-actions-bot-email-address/17204/6
-    - name: Set bot user data for commits
-      run: |
-        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
-        git config --global user.name "GitHub Actions Bot"
-
     # TODO: "grep certifi" should be removed following an upstream fix for: https://github.com/conda/conda/issues/11580
     - name: Update requirements-freeze.txt
       run: |
         source python/etc/profile.d/conda.sh
         conda activate venv_sct
         pip freeze | grep -v "-e git+" | grep -v "torch" | grep -v "certifi @ file" > requirements-freeze.txt
         conda deactivate
-        echo "# Platform-specific torch requirements (See SCT Issue #2745)" >> requirements-freeze.txt
+        echo "# Prioritize CPU version of torch (See https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3790#discussion_r888573886)" >> requirements-freeze.txt
         grep "torch" requirements.txt >> requirements-freeze.txt
+        echo "# Preserve platform-specific requirements missed by `pip freeze`" >> requirements-freeze.txt
```

Comment:
Oopsie... Using backticks to specify code/commands is a habit of mine with Markdown. 

But, here, the backticks evaluated `pip freeze` and stuck the output into the command. :rofl: 

![image](https://user-images.githubusercontent.com/16181459/178359645-3798b2bc-922a-4f0b-815a-135fbd3201a1.png)

```suggestion
        echo "# Preserve platform-specific requirements missed by 'pip freeze'" >> requirements-freeze.txt
```